### PR TITLE
Publishing content 23ww31

### PR DESCRIPTION
**Pull request by Maxim-Doronin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Changelog
+All notable changes to this project will be documented in this file.
+
+[1.1.0] - 2023-02-17
```
Comment (DariaMityagina) [inline CHANGELOG.md:4]: Maybe the date for this repository should be different, no?

**Review by DariaMityagina — APPROVED** [review]
LGTM 👍 
Thanks!

**Review by OrestChura — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* N/A
+
+## Removed
+* N/A
+
+## Fixed
+* N/A
+
+## Security
+* N/A
```
[the remark this answers is not in the document]
Comment (Maxim-Doronin) [reply CHANGELOG.md:23]: I'll ask the original code owners for active support on this file

**Pull request merged**